lib/fizzy/execute.cpp

```diff
@@ -495,12 +495,8 @@ inline bool invoke_function(const FuncType& func_type, const F& func, Instance&
 
     stack.drop(num_args);
 
-    const auto num_outputs = func_type.outputs.size();
```

Comment:
Should we leave in `assert(func_type.outputs.size() == ret.has_value)`? Or `assert(func_types.outputs.size() == (ret.has_value ? 1 : 0)` ?